sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -1281,44 +1322,30 @@ export class ManagementClient extends LinkEntity<RequestResponseLink> {
     throwTypeErrorIfParameterIsEmptyString(this._context.connectionId, "ruleName", ruleName);
 
     throwTypeErrorIfParameterMissing(this._context.connectionId, "filter", filter);
-    if (
-      typeof filter !== "boolean" &&
-      typeof filter !== "string" &&
-      !correlationProperties.some((validProperty) =>
-        isObjectWithProperties(filter, [validProperty])
-      )
-    ) {
+    if (!isSqlRuleFilter(filter) && !isCorrelationRuleFilter(filter)) {
```

Comment:
wait.. filter doesn't have to be an sql filter, right?
it could just be true/false also.

Reply:
true filter accepts all messages, which is the default.
And a false filter doesn't allow any message.

Reply:
Ohh, I see, we're calling it the sqlFilter.

Reply:
I see .NET has True and False filter, but I don't see it in our typing or `adminClient.createRule()`?  can you point me to an example of how we handle it?
```ts
export interface SqlRuleFilter {                                                                                                                                                                                          
  /**                                                                                                                                                                                                                     
   * SQL expression to use in the rule filter.                                                                                                                                                                            
   * Defaults to creating a true filter if none specified                                                                                                                                                                 
   */                                                                                                                                                                                                                     
  sqlExpression: string;                                                                                                                                                                                                  
                                                                                                                                                                                                                          
  /**                                                                                                                                                                                                                     
   * SQL parameters to the SQL expression in the rule filter.                                                                                                                                                             
   */                                                                                                                                                                                                                     
  sqlParameters?: { [key: string]: string | number | boolean };                                                                                                                                                           
} 
```

Reply:
> Ohh, I see, we're calling it the sqlFilter.

yeah I remove RuleDescription which has the `string` but is not used in public api surface. We don't support boolean in atom management client.